homeassistant/components/sensor/discogs.py

```diff
@@ -20,18 +21,20 @@
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_IDENTITY = 'identity'
-
 CONF_ATTRIBUTION = "Data provided by Discogs"
 
-DEFAULT_NAME = 'Discogs'
+NAME_COLLECTION_SENSOR = "Discogs Collection"
+NAME_WANTLIST_SENSOR = "Discogs Wantlist"
+NAME_RANDOM_RECORD = "Discogs Random Record"
 
-ICON = 'mdi:album'
+ICON_RECORD = 'mdi:album'
+ICON_PLAYER = 'mdi:record-player'
+UNIT_RECORDS = 'records'
 
-SCAN_INTERVAL = timedelta(hours=2)
+SCAN_INTERVAL = timedelta(minutes=10)
```

Comment:
Shortened the scan interval, I incorrectly thought I had to set a long interval for Discogs rate limiting but this is acceptable it seems.

Reply:
Would that mean that I'd have to create an object before the async_add_entities and pass that object to the sensors as a param:

```py

discogs_data = {
	'collection': discogs_client.identity().num_collection,
	'wantlist': discogs_client.identity().num_wantlist,
	...
}

async_add_entities([
	DiscogsCollectionSensor(data, name),
	DiscogsWantlistSensor(data, name),
	DiscogsRandomRecordSensor(data, name),
], True)
```